app/services/escort_creator.rb

```diff
@@ -54,4 +58,14 @@ def deep_clone_escort
       except: EXCEPT_GRAPH
     )
   end
+
+  def build_move_with_special_vehicle_details(escort)
```

Comment:
This is not consistent with the approach we use when cloning an existing escort. I would add the move to INCLUDE_GRAPH, and all the fields we are not interested in EXCEPT_GRAPH.

Reply:
Willdo